apis/python/src/tiledb/vector_search/index.py

```diff
@@ -77,7 +79,15 @@ def query(
 
         targets_m = array_to_matrix(np.transpose(targets))
 
-        r = query_vq(self._db, targets_m, k, nqueries, nthreads)
+        if query_type == "heap":
+            if nqueries != 10:
+                logging.warning("nqueries is ignored for heap query type")
```

Comment:
Should we remove `nqueries` from the params?

Reply:
Done, thanks